sqldelight-runtime/build.gradle

```diff
@@ -52,9 +53,11 @@ kotlin {
     fromPreset(presets.js, 'js')
     fromPreset(presets.iosX64, 'iosX64')
     fromPreset(presets.iosArm64, 'iosArm64')
+    fromPreset(presets.macosX64, 'macos')
+    fromPreset(presets.iosArm32, 'iosArm32')
```

Comment:
Most libraries for ios will also work fine with arm 32 and macos, so I generally add both. Native tests run on macos anyway, so good to add, although it makes builds take proportionally longer.